content/600-about/200-prisma-docs/20-style-guide/04-schema-models.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Docs example data model'
+metaTitle: 'Prisma docs style guide: example data model'
+metaDescription: 'This page describes the data models we use in our documentation examples.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+`User` and `Post` are the canonical models that we use in our examples throughout the Prisma docs.
+
+</TopBlock>
+
+## The `User` and `Post` data model
+
+We chose the `User` and `Post` models for the following reasons:
+
+- They do not require domain-specific knowledge.
+- They are commonly used as an example in the ORM space. This makes them familiar for users coming from other tools.
+- Consistent models make it easier for the reader when learning about different concepts, because there will be less context switching.
+- Less decision making and cognitive overhead for the docs authors. Using the same models reduces decision fatigue and is one less thing to worry about when trying to explain concepts.
+
+We plan to [add more schema examples](https://github.com/prisma/docs/issues/1626) to the docs. They will provide you with starting points for common data models, such as e-commerce.
```

Comment:
I feel we should remove this sentence... it might be along time before we can do that. Thoughts?

Reply:
I think that was there before I worked on the style guide. I don't like promising future features in docs, and the style guide should practice what it preaches, so I'm very happy to remove this!